tests/ui_tools/test_messages.py

```diff
@@ -36,7 +40,12 @@ def mock_external_classes(self, mocker, initial_index):
             ("private", [("email", ""), ("user_id", None)]),
         ],
     )
-    def test_init(self, mocker, message_type, set_fields):
+    def test_init(
+        self, 
+        mocker: MockerFixture, 
+        message_type: str, 
```

Comment:
For `message_type`, we likely want the new `MessageType` in `api_types.py`; that's also a string, but a more constrained type.